input/templates/my-account-address-book.json

```diff
@@ -22,8 +22,8 @@
 			"country" : "Country",
 			"edit" : "Edit"
 		},
-		"secondaryAddressTitle" : "Secondary Address",
-		"secondaryAddress" : {
+		"additionalAddressTitle" : "Additional Addresses",
```

Comment:
This should go to the i18n messages. I know it was already like this, but it was actually wrong. Everything static not coming from CTP should be contained as i18n messages.